lib/parallel_tests/gherkin/runner.rb

```diff
@@ -21,13 +21,16 @@ def run_tests(test_files, process_number, num_processes, options)
           options[:env] ||= {}
           options[:env] = options[:env].merge({ 'AUTOTEST' => '1' }) if $stdout.tty?
 
+          opts = cucumber_opts(options[:test_options])
+          puts "My opts are: #{opts}\n\n"
           cmd = [
             executable,
             (runtime_logging if File.directory?(File.dirname(runtime_log))),
-            cucumber_opts(options[:test_options]),
-            *sanitized_test_files
+            opts[0],
+            *sanitized_test_files,
+            opts[1]
```

Comment:
This line is needed in some way shape or form, because the `--retry n` flag must go at the end.

Reply:
why can't all options go to the end ?